src/Compilers/Core/CodeAnalysisTest/MetadataReferences/ModuleMetadataTests.cs

```diff
@@ -352,7 +352,10 @@ public unsafe void CreateFromUnmanagedMemoryStream_Prefetcha_LeaveOpenTrue(PEStr
             }
         }
 
-        [Fact]
+        /// <summary>
+        /// Only test in 64-bit process. <see cref="UnmanagedMemoryStream"/> throws if the given length is greater than the size of the available address space.
```

Comment:
Just adding some context here: we saw a random flaky test failure. Our only guess is if the address of the assembly were to be greater than 0x80000000, then adding 2GBish could wrap around.